<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>advancedOptions</class>
 <widget class="QDialog" name="advancedOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Advanced Options:</string>
   </property>
  </widget>
  <widget class="QLabel" name="description">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Advanced options may have very large impacts on the simulation output. Unless an option is checked and a valid value is supplied, a default one will be used.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="done">
   <property name="geometry">
    <rect>
     <x>482</x>
     <y>460</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>371</width>
     <height>341</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="3">
      <widget class="QToolButton" name="encounterHelp">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QToolButton" name="thetaHelp">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="diffLabel">
       <property name="text">
        <string>Animal Diffusion Rate:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="encounterLabel">
       <property name="text">
        <string>Encounter Rate:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="thetaLabel">
       <property name="text">
        <string>Theta:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="encounterValue">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="diffValue">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="thetaValue">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="diffEnable">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="encounterEnable">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="thetaEnable">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="diffHelp">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="sampleEnable">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="sampleLabel">
       <property name="text">
        <string>Diffusion Samples: (High performance impact)</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QToolButton" name="sampleHelp">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="sampleValue">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>181</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Option Information</string>
   </property>
   <widget class="QTextBrowser" name="explanationText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>diffEnable</tabstop>
  <tabstop>diffValue</tabstop>
  <tabstop>encounterEnable</tabstop>
  <tabstop>encounterValue</tabstop>
  <tabstop>thetaEnable</tabstop>
  <tabstop>thetaValue</tabstop>
  <tabstop>diffHelp</tabstop>
  <tabstop>encounterHelp</tabstop>
  <tabstop>thetaHelp</tabstop>
  <tabstop>done</tabstop>
  <tabstop>explanationText</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
